packages/components/src/internal/components/chart/ChartBuilderModal.tsx

```diff
@@ -0,0 +1,648 @@
+                                            type="checkbox"
+                                            checked={shared}
+                                            onChange={onToggleShared}
+                                        />{' '}
+                                        <span onClick={onToggleShared}>Make this chart available to all users</span>
+                                    </div>
+                                )}
+                            </div>
+                            <div className="col-xs-4 fields-col-left">
+                                {leftColFields.map(field => (
+                                    <div key={field.name}>
+                                        <label>
+                                            {field.label}
+                                            {field.required && ' *'}
+                                        </label>
+                                        <SelectInput
+                                            showLabel={false}
+                                            inputClass="col-xs-12"
+                                            placeholder="Select a field"
+                                            name={field.name}
+                                            options={getSelectOptions(model, selectedType, field)}
+                                            onChange={onSelectFieldChange}
+                                            value={fieldValues[field.name]?.value}
+                                        />
+                                    </div>
+                                ))}
+                            </div>
+                            <div className="col-xs-4 fields-col-right">
+                                {rightColFields.map(field => (
+                                    <Fragment key={field.name}>
+                                        <div>
+                                            <label>
+                                                {field.label}
+                                                {field.required && ' *'}
+                                            </label>
+                                            <SelectInput
+                                                showLabel={false}
+                                                inputClass="col-xs-12"
+                                                placeholder="Select a field"
+                                                name={field.name}
+                                                options={getSelectOptions(model, selectedType, field)}
+                                                onChange={onSelectFieldChange}
+                                                value={fieldValues[field.name]?.value}
+                                            />
+                                        </div>
+                                        {/* TODO change this to a select input with predefined options*/}
```

Comment:
Is this for a later PR or this PR?

Reply:
later PR, just using this display of the aggregate function for the time being until the next story